Adekoreday/Jur-Status
# Get All Status Functionality

>> alert ensure you have eslint and prettier vscode extension installed for code lint

### What This PR does

This PR enables admin on the blockchain to 

1. get all status in the contract

### What Changed In the codebase

1. Added more logic to the contract to enable the get functionality
2. Implement the allStatus component to render all status data

### How To test

```
truffle develop
>> migrate

cd ./app
npm run start:dev

```

Go to localhost:8080
- Authorize your account on metamask
- Add Status Type
- Create a new status based on the type added
- Click on the all status and see the all status page rendered 

### ScreenShot 

<img width="957" alt="allstatus" src="https://user-images.githubusercontent.com/44194416/103597615-fab9b180-4f00-11eb-8561-6eef3846fca0.PNG">
